libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -678,7 +678,7 @@ dictionary PrepareRefundResponse {
 dictionary RefundRequest {
     string swap_address;
     string to_address;
-    u32 sat_per_vbyte;
+    u32 sats_per_vbyte;
```

Comment:
`sat_per_vbyte` is probably fine. Not worth a breaking change on the `RefundRequest` and the `PrepareRefundRequest` imo.

Reply:
Good point. Fixed